integr8ly/workload-web-app
# DEL-329-Access AMQ console while app is running

**What**
Check if the AMQ console is accessible while the app is running

**To verify**
- Checkout this branch
- Run the below command 
```
export WORKLOAD_WEB_APP_IMAGE=quay.io/<replace-with-your-repo>/workload-app
export GRAFANA_DASHBOARD=true
make local/build-deploy
```
- wait for the app to get deployed
- Login to the grafana and verify if the `AMQ Console` panel is present in workload-app dashboard

**TODO**
- Pass through the OAuth proxy to access the AMQ Console URL
